app/errors.py

```diff
@@ -0,0 +1,17 @@
+class VaccineError(Exception):
+    pass
+
+
+class NotVaccinatedError(VaccineError):
+    def __init__(self, message: str = "NotVaccinatedError") -> None:
```

Comment:
The default message for `NotVaccinatedError` could be more descriptive. Consider using a message that provides more context, such as 'Visitor is not vaccinated.'